gitflow_toolbox/ensure_mr.py

```diff
@@ -5,31 +5,73 @@
 
 
 @click.command()
-@click.option("--remote/--current", default=False)
 @click.argument("source_branch", type=str)
 @click.argument("target_branch", type=str)
 @click.argument("title", type=str)
 @click.argument("description", type=str)
 @click.argument("labels", type=str, nargs=-1)
+@click.option(
+    "--keep-source-branch",
+    "-ksb",
+    is_flag=True,
+    help="Flag used to specify that you want to keep the source branch once the merge request has been accepted",
+)
+@click.option(
+    "--squash",
+    "-s",
+    is_flag=True,
+    help="Flag used to specify that you want to squash commit when the merge request is accepted."
+    " This is an option of Gitlab which could be manually deactivated by user after the merge request creation.",
+)
+@click.option(
+    "--remote/--current",
+    "-r/-c",
+    default=False,
+    help="Flag which allow you to chose between configured target or remote to run this tool."
+    " By default, the script use --current flag.",
+)
+@click.option(
+    "--assignee_ids",
+    "-ai",
```

Comment:
Use a single letter as [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
```suggestion
    "-a",
```

Reply:
To keep same logic as reviewer_ids